CMakeLists.txt

```diff
@@ -2,24 +2,32 @@ cmake_minimum_required(VERSION 3.5.0)
 project(opw_kinematics VERSION 0.1.0 LANGUAGES CXX)
 
 find_package(Eigen3 REQUIRED)
+find_package(cmake_common_scripts REQUIRED)
+
+add_code_coverage_all_targets()
+
 if(NOT EIGEN3_INCLUDE_DIRS)
   set(EIGEN3_INCLUDE_DIRS ${EIGEN3_INCLUDE_DIR})
 endif()
```

Comment:
What is this? You're allowed to overwrite the version of Eigen found by CMake?

Reply:
I believe this is just making sure the variable EIGEN3_INCLUDE_DIRS exists and if not it sets it to EIGEN3_INCLUDE_DIR. They should be the same but I believe older versions may have only had EIGEN3_INCLUDE_DIR which was an error in Eigen.